psi4/driver/procedures/proc.py

```diff
@@ -2594,7 +2594,7 @@ def run_detci_property(name, **kwargs):
         core.print_out("\n   ===> %s properties for all CI roots <=== \n\n" % name.upper())
         for root in states:
             oe.set_title("%s ROOT %d" % (name.upper(), root))
-            root = root - 1
+            # root = root - 1
```

Comment:
Is there a comment in detci somewhere about external/internal root numbering? Are they equivalent now?